### DISCUSSION TITLE
Can laravel reveb be used?

### DISCUSSION THREAD

**Discussion by mr-ghomi** [NativePHP/laravel#302] [Q&A] [answered]
Can laravel reveb be used?

windows 11
laravel 11
laravel reverb

**Comment by simonhamp** [comment]
@mr-ghomi Not yet, but [I'm working on it](https://github.com/NativePHP/electron-plugin/pull/27)

**Comment by simonhamp** [comment, marked as answer]
@mr-ghomi we've side-stepped setting up a websocket server altogether! You can get real-time feedback in your app using plain Javascript.

Please read through the [Broadcasting](https://nativephp.com/docs/1/digging-deeper/broadcasting#listening-with-javascript) section of the docs for more details.
